sahi/utils/mot.py

```diff
@@ -250,26 +250,31 @@ def _create_frame_symlinks(self, coco_images_dir: str, export_dir: str):
             if Path(mot_frame.file_name).suffix == "":
                 print(f"image file has no suffix, skipping it: '{mot_frame.file_name}'")
                 return
-            elif Path(mot_frame.file_name).suffix not in [".jpg", ".jpeg"]:  # TODO: extend this list
+            elif Path(mot_frame.file_name).suffix not in [".jpg", ".jpeg", ".bmp", ".gif", ".png", ".tiff"]:
                 print(f"image file has incorrect suffix, skipping it: '{mot_frame.file_name}'")
                 return
-            # set coco and mot image paths
-            coco_img_path_tmp = os.path.join(coco_images_dir, mot_frame.file_name)
+            # set source and mot image paths
             suffix = Path(mot_frame.file_name).suffix
 
-            if Path(coco_img_path_tmp).is_file():
-                coco_image_path = os.path.abspath(coco_img_path_tmp)
+            if not mot_frame.file_name:
+                raise TypeError(f"mot_frame.file_name expected to be string but got: {type(mot_frame.file_name)}")
+            elif os.path.isabs(mot_frame.file_name):
+                if not Path(mot_frame.file_name).is_file():
+                    raise ValueError(f"there is not image file in path: {str(Path(mot_frame.file_name))}")
```

Comment:
Gramer olarak doğru olması için şu ikisinden biri ile değiştir :)
- there is no image file in path
- there is not any image file in path